book.rb

```diff
@@ -0,0 +1,14 @@
+class Book
+  attr_accessor :title, :author
+  attr_reader :rental
```

Comment:
- [ ] `rental` should be marked with `attr_accessor`, as we will be both _accessing_ it as well as _modifying_ it. Kindly ensure to improve upon this aspect.